utils_test.py

```diff
@@ -16,6 +16,7 @@
 
 # pylint: disable=relative-import
```

Comment:
Ah, sorry didn't know that. to be consistent follow the old one. We can refactor later anyway.